src/components/nodes/sequenceFlow/sequenceFlow.vue

```diff
@@ -68,6 +68,18 @@ export default {
         });
       },
     },
+    defaultFlow: {
+      get() {
+        return this.shape.attr('line').sourceMarker['stroke-width'] > 0;
+      },
+      set(value) {
+        this.shape.attr('line', {
+          sourceMarker: {
+            'stroke-width': value ? 2 : 0,
+          },
+        });
+      },
+    },
```

Comment:
If you get rid of the getter then this can be its own method, removing the need for the computed property.

Reply:
Computed removed, changed to a `setDefaultMarker` method